Sources/MockoloFramework/Templates/ConcurrencyHelpersTemplate.swift

```diff
@@ -0,0 +1,45 @@
+func applyConcurrencyHelpersTemplate() -> String {
+    return #"""
+fileprivate func warnIfNotSendable<each T>(function: String = #function, _: repeat each T) {
+    print("At \(function), the captured arguments are not Sendable, it is not concurrency-safe.")
+}
```

Comment:
This function is called only if there is an issue with the user's implementation, so it can be resolved if the user fixes the problem.

As an escape hatch, I want to make it possible to disable it with:
```
/// @mockable(history: fooFunc = false)
```
(Probably it cannot be done now.)